src/components/panel/panel.js

```diff
@@ -173,8 +173,9 @@ angular
 
 
 /**
- * @ngdoc object
+ * @ngdoc type
```

Comment:
Is the end result here to put a whole new section for Types in the sidenav, or just add this these docs to the `$mdPane` page? (the former would be desirable)